com.corifeus.onenote.appdata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop-application">com.corifeus.onenote</id>
+  <metadata_license>FSFAP</metadata_license>
+  <name>P3X OneNote</name>
+  <summary>A Linux compatible version of OneNote</summary>
+  <description>
+    <p>This is a Liunx OneNote helper program.
+For verbose description refer to the GitHub below...</p>
+
+    <p><a href="https://github.com/patrikx3/onenote">github.com/patrikx3/onenote</a></p>
+  </description>
+  <launchable type="desktop-id">com.corifeus.onenote.desktop</launchable>
+  <categories>
```

Comment:
I believe you don't need the `<categories>` tag here once they are specified in the `.desktop` file.